src/vue/forms/VerificationCorporateForm.vue

```diff
@@ -0,0 +1,259 @@
+<template>
+  <div class="verification-corporate-form">
+    <p class="verification-corporate-form__account-info-title">
+      {{ 'verification-form.account-information-lbl' | globalize }}
+    </p>
+    <form
+      novalidate
+      class="app-form"
+      @submit.prevent="submit">
```

Comment:
if you press enter on any field the form will try to submit itself without any `FormConfirmation` showing. if you click the button below the `FormConfirmation` appears. leave only action on `@submit.prevent` and move the logic from `@click.prevent` to `@submit.prevent`

Reply:
Resolved, fixed